web/src/lib/core/auth/providers/userapikey/UserAPIKeyAuthProviderClient.ts

```diff
@@ -1,57 +1,65 @@
-import { UserAPIKey, StitchAuthRequestClient, StitchAuthRoutes, StitchRequestClient } from "stitch-core";
+import {
+  UserAPIKey,
+  StitchAuthRequestClient,
+  StitchAuthRoutes,
+  StitchRequestClient
+} from "stitch-core";
 import { ObjectID } from "bson";
 import AuthProviderClientFactory from "../internal/AuthProviderClientFactory";
 import UserAPIKeyAuthProviderClientImpl from "./internal/UserAPIKeyAuthProviderClientImpl";
 
 export interface UserAPIKeyAuthProviderClient {
   /**
    * Creates a user API key that can be used to authenticate as the current user.
-   * 
+   *
    * @param name the name of the API key to be created.
    */
-  createApiKey(name: string): Promise<UserAPIKey>
+  createApiKey(name: string): Promise<UserAPIKey>;
 
   /**
    * Fetches a user API key associated with the current user.
-   * 
+   *
    * @param keyId the id of the API key to fetch.
    */
-  fetchApiKey(keyId: ObjectID): Promise<UserAPIKey>
+  fetchApiKey(keyId: ObjectID): Promise<UserAPIKey>;
 
   /**
    * Fetches the user API keys associated with the current user.
    */
-  fetchApiKeys(): Promise<UserAPIKey[]>
+  fetchApiKeys(): Promise<UserAPIKey[]>;
 
   /**
    * Deletes a user API key associated with the current user.
-   * 
+   *
    * @param keyId the id of the API key to delete
    */
-  deleteApiKey(keyId: ObjectID): Promise<void>
+  deleteApiKey(keyId: ObjectID): Promise<void>;
 
   /**
    * Enables a user API key associated with the current user.
-   * 
+   *
    * @param keyId the id of the API key to enable
    */
-  enableApiKey(keyId: ObjectID): Promise<void>
+  enableApiKey(keyId: ObjectID): Promise<void>;
 
   /**
    * Disables a user API key associated with the current user.
-   * 
+   *
    * @param keyId the id of the API key to disable
    */
-  disableApiKey(keyId: ObjectID): Promise<void>
+  disableApiKey(keyId: ObjectID): Promise<void>;
 }
 
 export namespace UserAPIKeyAuthProviderClient {
-  export const Factory: AuthProviderClientFactory<UserAPIKeyAuthProviderClient> =
-  new class implements AuthProviderClientFactory<UserAPIKeyAuthProviderClient> {
+  export const Factory: AuthProviderClientFactory<
+    UserAPIKeyAuthProviderClient
```

Comment:
[nit] another weirdly formatted generic

Reply:
This may be due to a character limit on the line. I'll look into it, but I'm not going to spend that much time for it. It's not _that_ ugly